From 6acaf73968793201e2af197ef030a708d1d9f003 Mon Sep 17 00:00:00 2001
From: Mohsen Dolaty <mohsen.dolaty@amd.com>
Date: Thu, 8 Feb 2024 11:39:25 -0600
Subject: [PATCH 1/1] ARM dts aspeed: Genoa Add MON_RSMRST_L to GPIO G7

Add GPIO G7 MON_RSMRST_L to all Genoa platform to help with
applying the Power Restore Policy during BMC initialization.

Signed-off-by: Mohsen Dolaty <mohsen.dolaty@amd.com>
---
 arch/arm/boot/dts/aspeed-bmc-amd-onyx-i3c.dts     | 2 +-
 arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts         | 2 +-
 arch/arm/boot/dts/aspeed-bmc-amd-quartz-i3c.dts   | 2 +-
 arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts       | 2 +-
 arch/arm/boot/dts/aspeed-bmc-amd-ruby-i3c.dts     | 2 +-
 arch/arm/boot/dts/aspeed-bmc-amd-ruby.dts         | 2 +-
 arch/arm/boot/dts/aspeed-bmc-amd-titanite-i3c.dts | 2 +-
 arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts     | 2 +-
 8 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-onyx-i3c.dts b/arch/arm/boot/dts/aspeed-bmc-amd-onyx-i3c.dts
index 081ea6bb3731..c9f6c33fc764 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-onyx-i3c.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-onyx-i3c.dts
@@ -787,7 +787,7 @@ &gpio0 {
 	/*D0-D7*/	"","","","","","","","",
 	/*E0-E7*/	"","","","","","","","",
 	/*F0-F7*/	"","","","","","","","",
-	/*G0-G7*/	"","","","","","","","",
+	/*G0-G7*/	"","","","","","","","MON_RSMRST_L",
 	/*H0-H7*/	"","ASSERT_WARM_RST_BTN_L","ASSERT_SOC_RST_BTN_L","","","","","",
 	/*I0-I7*/	"","","","","","","","P0_I3C_APML_ALERT_L",
 	/*J0-J7*/	"","","","","","","","",
diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts b/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
index b3d4781d3435..a3c344192112 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
@@ -801,7 +801,7 @@ &gpio0 {
 	/*D0-D7*/	"","","","","","","","",
 	/*E0-E7*/	"","","","","","","","",
 	/*F0-F7*/	"","","","","","","","",
-	/*G0-G7*/	"","","","","","","","",
+	/*G0-G7*/	"","","","","","","","MON_RSMRST_L",
 	/*H0-H7*/	"","ASSERT_WARM_RST_BTN_L","ASSERT_SOC_RST_BTN_L","","","","","",
 	/*I0-I7*/	"","","","","","","","P0_I3C_APML_ALERT_L",
 	/*J0-J7*/	"","","","","","","","",
diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-quartz-i3c.dts b/arch/arm/boot/dts/aspeed-bmc-amd-quartz-i3c.dts
index 1e4bfbcba6b1..822543d57995 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-quartz-i3c.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-quartz-i3c.dts
@@ -1101,7 +1101,7 @@ &gpio0 {
 	/*D0-D7*/	"","","","","","","","",
 	/*E0-E7*/	"","","","","","","","",
 	/*F0-F7*/	"","","","","","","","",
-	/*G0-G7*/	"","","","","","","","",
+	/*G0-G7*/	"","","","","","","","MON_RSMRST_L",
 	/*H0-H7*/	"","ASSERT_WARM_RST_BTN_L","ASSERT_SOC_RST_BTN_L","","","","","",
 	/*I0-I7*/	"","","","","","","","P0_I3C_APML_ALERT_L",
 	/*J0-J7*/	"","","","","","","","",
diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts b/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts
index 159db8ecc351..6d1ec3354f80 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts
@@ -1095,7 +1095,7 @@ &gpio0 {
 	/*D0-D7*/	"","","","","","","","",
 	/*E0-E7*/	"","","","","","","","",
 	/*F0-F7*/	"","","","","","","","",
-	/*G0-G7*/	"","","","","","","","",
+	/*G0-G7*/	"","","","","","","","MON_RSMRST_L",
 	/*H0-H7*/	"","ASSERT_WARM_RST_BTN_L","ASSERT_SOC_RST_BTN_L","","","","","",
 	/*I0-I7*/	"","","","","","","","P0_I3C_APML_ALERT_L",
 	/*J0-J7*/	"","","","","","","","",
diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-ruby-i3c.dts b/arch/arm/boot/dts/aspeed-bmc-amd-ruby-i3c.dts
index abdd8ed929db..23e9c0fdedc9 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-ruby-i3c.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-ruby-i3c.dts
@@ -841,7 +841,7 @@ &gpio0 {
 	/*D0-D7*/	"","","","","","","","",
 	/*E0-E7*/	"","","","","","","","",
 	/*F0-F7*/	"","","","","","","","",
-	/*G0-G7*/	"","","","","","","","",
+	/*G0-G7*/	"","","","","","","","MON_RSMRST_L",
 	/*H0-H7*/	"","ASSERT_WARM_RST_BTN_L","ASSERT_SOC_RST_BTN_L","","","","","",
 	/*I0-I7*/	"","","","","","","","P0_I3C_APML_ALERT_L",
 	/*J0-J7*/	"","","","","","","","",
diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-ruby.dts b/arch/arm/boot/dts/aspeed-bmc-amd-ruby.dts
index c16c6276dca3..5db84002e2c0 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-ruby.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-ruby.dts
@@ -841,7 +841,7 @@ &gpio0 {
 	/*D0-D7*/	"","","","","","","","",
 	/*E0-E7*/	"","","","","","","","",
 	/*F0-F7*/	"","","","","","","","",
-	/*G0-G7*/	"","","","","","","","",
+	/*G0-G7*/	"","","","","","","","MON_RSMRST_L",
 	/*H0-H7*/	"","ASSERT_WARM_RST_BTN_L","ASSERT_SOC_RST_BTN_L","","","","","",
 	/*I0-I7*/	"","","","","","","","P0_I3C_APML_ALERT_L",
 	/*J0-J7*/	"","","","","","","","",
diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-titanite-i3c.dts b/arch/arm/boot/dts/aspeed-bmc-amd-titanite-i3c.dts
index 095bdb2bc482..7efa65e53a4b 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-titanite-i3c.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-titanite-i3c.dts
@@ -877,7 +877,7 @@ &gpio0 {
 	/*D0-D7*/	"","","","","","","","",
 	/*E0-E7*/	"","","","","","","","",
 	/*F0-F7*/	"","","","","","","","",
-	/*G0-G7*/	"","","","","","","","",
+	/*G0-G7*/	"","","","","","","","MON_RSMRST_L",
 	/*H0-H7*/	"","ASSERT_WARM_RST_BTN_L","ASSERT_SOC_RST_BTN_L","","","","","",
 	/*I0-I7*/	"","","","","","","","P0_I3C_APML_ALERT_L",
 	/*J0-J7*/	"","","","","","","","",
diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts b/arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts
index dcdbd72fd54c..f87339f6c2a1 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts
@@ -871,7 +871,7 @@ &gpio0 {
 	/*D0-D7*/	"","","","","","","","",
 	/*E0-E7*/	"","","","","","","","",
 	/*F0-F7*/	"","","","","","","","",
-	/*G0-G7*/	"","","","","","","","",
+	/*G0-G7*/	"","","","","","","","MON_RSMRST_L",
 	/*H0-H7*/	"","ASSERT_WARM_RST_BTN_L","ASSERT_SOC_RST_BTN_L","","","","","",
 	/*I0-I7*/	"","","","","","","","P0_I3C_APML_ALERT_L",
 	/*J0-J7*/	"","","","","","","","",
--
2.25.1
